lighthouse-core/test/audits/byte-efficiency/offscreen-images-test.js

```diff
@@ -128,13 +183,51 @@ describe('OffscreenImages audit', () => {
     assert.equal(auditResult.items.length, 4);
   });
 
+  it('passes images with a specified loading attribute', async () => {
```

Comment:
nit: Check the opposite of this test to make sure that arbitrary loading attributes don't pass?

```javascript
it('fails images with a non-lazy loading attribute', async () => {
...
    const artifacts = {
      ViewportDimensions: DEFAULT_DIMENSIONS,
      ImageElements: [
          ...
          loading: 'not-lazy',
          ...
        }),
...
```